doc/config.toml.tmpl

```diff
@@ -25,6 +25,14 @@ pygmentsUseClasses = true
   github_repository = "https://github.com/TheThingsNetwork/lorawan-stack"
   github_repository_edit = "https://github.com/TheThingsNetwork/lorawan-stack/edit/master/doc/content"
   version = "{{.CurrentVersion}}"
+  [params.intercom]
+    enabled = false
+    appid = ""
```

Comment:
Is this really an Appid? Or should it be `app_id`?

Reply:
it's an app_id but that that doesn't seem to work because hugo thinks `HUGO_PARAMS_INTERCOM_APP_ID`is
```
[params]
[params.intercom]
[params.intercom.app]
id
```